src/client/components/app/LineDetailsDisplay/LineMetadata.tsx

```diff
@@ -294,12 +376,17 @@ export const LineMetadata = () => {
     }
   };
 
-  const formattedCollateralData = allCollateral.map((c) => ({
-    ...c,
-    key: c.type + c.token.toString(),
-    align: 'flex-start',
-    actions: getCollateralRowActionForRole(userPositionMetadata.role),
-  }));
+  const formattedCollateralData = allCollateral.map((c) => {
+    const tokenIcon = Object.keys(tokensMap).length > 0 ? tokensMap[getAddress(c.token.address)]?.icon ?? '' : '';
```

Comment:
not making this change, need both because cannot retrieve icon from `tokensMap` if `tokensMap` is undefined. 

changing the logic to be more explicit than we're checking if `tokensMap` is undefined